airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -823,18 +827,41 @@ def __init__(
         config: dict | None = None,
         wait_for_completion: bool = True,
         aws_conn_id: str = "aws_default",
-        waiter_countdown: int = 25 * 60,
-        waiter_check_interval_seconds: int = 60,
+        waiter_max_attempts: int | None | ArgNotSet = NOTSET,
+        waiter_delay: int | None | ArgNotSet = NOTSET,
```

Comment:
why allow None here ? It's either set to an integer value or not set, isn't it ?
(same question for other constructors below)